reference/tools/gnu/pkgconfigdeps.rst

```diff
@@ -122,9 +124,30 @@ also have a **regular require**. Solving this conflict is specially important wh
 cross-building because the tool (that will run in the building machine) belongs to a
 different binary package than the library, that will "run" in the host machine.
 
-You can use the **build_context_suffix** attribute to specify a suffix for a requirement,
-so the files/requires/names of the requirement in the build context (tool require) will be
-renamed:
+You can use the ``build_context_folder`` attribute to specify a folder to save the `*.pc` files created by all those
+build requirements listed in the ``build_context_activated`` one:
+
+.. code-block:: python
+
+    tool_requires = ["my_tool/0.0.1"]
+    requires = ["my_tool/0.0.1"]
+    def generate(self):
+        pc = PkgConfigDeps(self)
+        # generate the *.pc file for the tool require
+        pc.build_context_activated = ["my_tool"]
+        # save all the *.pc files coming from the "my_tool" build context and its requirements
+        pc.build_context_folder = "build"  # [generators_folder]/build/
+        pc.generate()
+
+
+
+build_context_suffix
+^^^^^^^^^^^^^^^^^^^^
+
+*DEPRECATED: use build_context_folder attribute instead*
```

Comment:
It is still not very clear that the deprecation of this and the recommendation to use ``build_context_suffix`` is the way to go also for autotools.  If it is not, maybe we should recommend this for Meson only?

Maybe we need to understand the potential autotools usages of the ``build_context_suffix`` and ``build_context_folder``